### PR TITLE
Added a marker to denote primary specimen to document viewer.

### PR DESCRIPTION
**Pull request by MLumme**
The marker should always stay next to the first row of the unit. Testable at https://183999568.dev.laji.fi/view?uri=http:%2F%2Fid.luomus.fi%2FJA.1122334456

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 FROM node:14
 
 # set working directory & user
-USER node
+# USER node
```
Comment (olzraiti) [inline Dockerfile:5]: Is this committed on purpose?

File: projects/laji/src/app/shared-modules/document-viewer/unit-rows/unit-rows.component.html [hunk 2]
```diff
@@ -1,15 +1,34 @@
+<ng-template #primarySpecimenHideFacts>
+  <ng-container *ngIf="!showFacts then primarySpecimen"></ng-container>
+</ng-template>
+<ng-template #primarySpecimen>
+  <ng-container *ngIf="unit.primarySpecimen">
+    <div class="primary-specimen">{{"result.unit.primarySpecimen" | translate}}</div>
```
Comment (olzraiti) [inline projects/laji/src/app/shared-modules/document-viewer/unit-rows/unit-rows.component.html:6]: use white space in `{{ }}`

**Review by olzraiti — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (MLumme) [reply Dockerfile:5]: It definitely is not supposed to be there, it's just to get the thing to run locally in docker, but I must've been careless when adding the files.

Comment (MLumme) [reply Dockerfile]: Both should be fixed now in the latest commit.

**Review by olzraiti — APPROVED** [review]
Thanks!

**Pull request merged**